packages/integrations/lit/client-shim.js

```diff
@@ -7,14 +7,8 @@ async function polyfill() {
 	});
 }
 
-const polyfillCheckEl = new DOMParser()
-	.parseFromString(
-		`<p><template shadowroot="open" shadowrootmode="open"></template></p>`,
-		'text/html',
-		{
-			includeShadowRoots: true,
-		}
-	)
+const polyfillCheckEl = Document
+	.parseHTMLUnsafe(`<p><template shadowrootmode="open"></template></p>`)
```

Comment:
Hi,

May I ask about the comparability? 

`parseFromString` is there in basically every browser (https://developer.mozilla.org/en-US/docs/Web/API/DOMParser/parseFromString) while `parseHTMLUnsafe` is not yet we'll supported (https://caniuse.com/mdn-api_document_parsehtmlunsafe_static today reports global of just under 58%)

Wont that be a problem?